newrelic-php-agent/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: Newrelic PHP Agent
 name: newrelic-php-agent
-version: 0.1.0
\ No newline at end of file
+version: 0.1.1
```

Comment:
@cw-niinuma 
Please add `appVersion`.
It ’s easier to see if we ’re supporting an application version

Reply:
@cw-ozaki okay, I understand.

Reply:
@cw-ozaki This commit was supported.
https://github.com/chatwork/charts/pull/118/commits/4c8a799ff467628d06b15e94d6462febfc03c210